stream-parent/stream-core/src/main/java/org/interledger/stream/frames/StreamFrameConstants.java

```diff
@@ -0,0 +1,21 @@
+package org.interledger.stream.frames;
+
+public interface StreamFrameConstants {
```

Comment:
Unit test coverage tracking ticket in #245

Reply:
Resolved in 0601a43